UpdateTrainingForm/UpdateTraining.js

```diff
@@ -0,0 +1,36 @@
+import axios from "../../../../api/axios";
+import { TRAININGS_URL } from "../../../../Utils/URLConstants";
+export async function UpdateTraining (trainingInfo, timeRange, training){
+        try{
+            const startTime = timeRange[0];
+            const endTime = timeRange[1];
+            const trainingType= trainingInfo.trainingType;
+            const email= trainingInfo.email;
+            const price= trainingInfo.price;
+            const day= trainingInfo.day;
+            const description= trainingInfo.description;
+            const slots= trainingInfo.slots;
+            const trainingId= training._id;
+            const response = await axios.put(TRAININGS_URL+'/'+training._id,
```

Comment:
How can you check duplication here? What if the same name and time already exist?
Do you do validation for each input?

Reply:
The trainer can decide to have a training at the same time. Maybe if it's online he/she would want to connect to two devices. It's their choice. I don't see any reason why I should add that restriction